### PR TITLE
Add detail page that links to add liquidity.

### PR DESCRIPTION
**Pull request by TBomberman**
image
We can finalize the columns later.
1. Renamed route to deposit from addliquidity
image
2. Used dynamic routing to feed address to deposit page (add liquidity)
3. Changed Lend and Open Loan title to Provide Liquidity and Short Liquidity as per Devin
4. Short Liquidity to be added after Open Loan work.
ty @Pochetes or @dam417

### CODE REVIEW

File: pages/positions/options.tsx [hunk 1]
```diff
@@ -21,13 +21,13 @@ const Options: NextPage = () => {
       <div className={style.optionsContainer}>
         <h1 className={style.optionsHeader}>Open a New Position</h1>
         <div className={style.optionsButtonContainer}>
-          <Link href={'addliquidity'}>
+          <Link href={'deposit/0'}>
```
Comment (Pochetes) [inline pages/positions/options.tsx:24]: where does this link go? does it still go to `addLiquidity`

File: pages/positions/options.tsx [hunk 2]
```diff
@@ -21,13 +21,13 @@ const Options: NextPage = () => {
...
             <button className={style.optionsButtonContent}>
               <span className={style.hoveredOptionsButtonContent}></span>
               <div className={style.optionsButtonPrimary}>
-                <div className={style.buttonTitle}>Lend</div>
+                <div className={style.buttonTitle}>Provide Liquidity</div>
                 <div className={style.buttonText}>
-                  Provide liquidity with any ERC-20 Token pair.
+                  Lend liquidity with any ERC-20 Token pair.
```
Comment (Pochetes) [inline pages/positions/options.tsx:30]: I think we need to change this to a more playful message. I think it's just saying the same thing as the title. Let's run this by Devin

File: pages/positions/options.tsx [hunk 3]
```diff
@@ -41,9 +41,9 @@ const Options: NextPage = () => {
             <button className={style.optionsButtonContent}>
             <span className={style.hoveredOptionsButtonContent}></span>
               <div className={style.optionsButtonPrimary}>
-                <div className={style.buttonTitle}>Borrow</div>
+                <div className={style.buttonTitle}>Short Liquidity</div>
                 <div className={style.buttonText}>
-                  Open a loan if you're long volatility and hedge against impermanent loss.
+                  Borrow liquidity or open a loan if you're long volatility and hedge against impermanent loss.
```
Comment (Pochetes) [inline pages/positions/options.tsx:46]: As well as with this one.

**Review by Pochetes — APPROVED** [review]
LGTM, just left some comments. Just keep in mind that we would need to switch from a modal to an actual page.

File: pages/positions/options.tsx (hunk 1 above)
Comment (TBomberman) [reply pages/positions/options.tsx:24]: yes, the address will be 0, it needs to have some value

File: pages/positions/options.tsx (hunk 2 above)
Comment (TBomberman) [reply pages/positions/options.tsx:30]: changed wording as per Devin

File: pages/positions/options.tsx (hunk 3 above)
Comment (TBomberman) [reply pages/positions/options.tsx]: changed wording as per Devin

**Pull request merged**